### PR TITLE
Use light DOM font size for non-headline slot

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/26566

This changes the `font-size` to 1.6rem for the slotted content.
## Testing done

Local stencil dev server

### Before

image

### After

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/components/va-alert/va-alert.css [hunk 1]
```diff
@@ -41,8 +41,9 @@ div.body {
   vertical-align: middle;
   width: 100%;
 }
-::slotted(:not([slot])){
+::slotted(:not([slot])) {
   margin-top: 2rem;
+  font-size: 1.6rem;
```
Comment (cvalarida) [inline src/components/va-alert/va-alert.css:46]: Why do this instead of removing the `font-size: 2rem` in the `:host` selector?

Comment (bkjohnson) [reply src/components/va-alert/va-alert.css:46]: I'll see what that looks like :+1:

Comment (bkjohnson) [reply src/components/va-alert/va-alert.css:46]: By having the `font-size: 2rem` in the `:host` selector along with `font-size: 1em` in the `::slotted` selector we ensure that `h4` and other headers in the `headline` slot appear as 2rem regardless of what they are styled as in the light DOM.

Comment (cvalarida) [reply src/components/va-alert/va-alert.css:46]: Ahh, I see. If that's what we're aiming for, I'd recommend styling for just that specifically.
```css
h1, h2, h3, h4, h5, h6 {
  font-size: 2rem
}
```

File: src/components/va-alert/va-alert.css [hunk 2]
```diff
@@ -13,7 +12,7 @@
 }
 
 ::slotted([slot='headline']) {
-  font-size: 1em !important;
+  font-size: 2rem !important;
```
Comment (bkjohnson) [inline src/components/va-alert/va-alert.css:15]: It looks like this does the trick.

Testing sample:

```jsx
        <va-alert status="success">
          <h4 slot="headline">I am a level 4 headline</h4>
          <div>Content blah blah blah</div>
        </va-alert>
        <br />
        <va-alert status="success">
          <h3 slot="headline">Congrats, you did it!</h3>
          <div>Content blah blah blah</div>
        </va-alert>
        
        <AlertBox headline="Testing" status="info">
          <div>Some more content</div>
        </AlertBox>
 ```

In firefox:

image


In IE11:

signal-2021-07-02-114443

**Review by cvalarida — APPROVED** [review]

**Pull request merged**